docs/input-kafka.asciidoc

```diff
@@ -2,8 +2,8 @@
 :plugin: kafka
 :type: input
 :default_codec: plain
-:kafka_client: 2.4
```

Comment:
Thanks for updating these vars. I'm happy to seem them being used. :-)

We used version `2.5.1` in the integration doc and version `2.5` for both input and output docs. We should be consistent.  Perhaps we should have your version checking in place before we commit to maintaining more granular versions (2.5.1 instead of 2.5).

Reply:
The idea here is we need to have the precise version somewhere (to be able to know bug fixes etc).

For the input/output specific documentation it seemed okay to only have the minor version compatibility (the Kafka docs linked only uses minor version paths such as `/25/...`) but I am fine changing that if you prefer so.

I do have the version checking PR as a follow-up on this one.

Reply:
I'm fine with merging as is, and waiting for your version checking PR to go more granular on version info in input and output.  I have more changes I need to make anyway, so I can make the fixes after version checking is merged. Thanks!